### DISCUSSION TITLE
How to convince the company admins to grant Presencelight access to the MS Graph API

### DISCUSSION THREAD

**Discussion by smatthew-vitek** [isaacrlevin/presencelight#877] [Q&A] [answered]
Hi Isaac,

Thanks for creating this nice productivity enhancing app! Hopefully you can help me out here getting it actually up and running.

I've requested our security department to grant the app the access it requires, since that's apparantly what the app needs (see screenshots below). From the documentation, the [MS page on the Graph API](https://learn.microsoft.com/nl-nl/graph/api/presence-get?view=graph-rest-1.0&tabs=http) mentions getting presence for yourself or others.
It's important for the admins to know that they're not introducing a potential security breach just because I'm keen on communicating me being busy or not through Philips Hue lights 😄 

Thanks for your time and regards, Vikaash

![image](https://github.com/isaacrlevin/presencelight/assets/157807975/1c43704a-3ead-4c9c-a10d-4209b0f12d08)

![image](https://github.com/isaacrlevin/presencelight/assets/157807975/e553a6ab-8577-4d4d-b97a-ec225d4ee7d7)

![image](https://github.com/isaacrlevin/presencelight/assets/157807975/d1b8fe27-1e52-4f08-b9d7-c8f80186cc16)

![image](https://github.com/isaacrlevin/presencelight/assets/157807975/420a2dfe-5182-4982-a6f2-b2572a8ce11e)

**Comment by isaacrlevin** [comment, marked as answer]
How can I help? The scope of this app is to get Presence, Profile and Profile Image for the currently logged in user, there is no delegation at all?
